index.html

```diff
@@ -0,0 +1,612 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<link href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+	<link href="index.css" rel="stylesheet">
+	<title>New York times</title>
+</head>
+<body>
+	<header>
+		<nav class="nav-bar">
+			<div class="left-nav-bar">
+				<i class="nav-icon fas fa-bars"><a></a></i> <i class="fas fa-search"></i> <span class="space-cosmos"><a href="https://www.nytimes.com/section/science/space">SPACE & COSMOS</a></span>
+			</div>
+			<div class="center-nav-bar">
+				<svg height="25" width="185" xmlns="http://www.w3.org/2000/svg">
```

Comment:
- [ ]  put height and width selectors in a class or Id in your CSS file as no inline styling is permitted in the HTML